src/Settings.php

```diff
@@ -39,6 +39,12 @@ class Settings
     public $phpExecutable = 'php';
 
     /**
+     * Git executable
+     * @var string
+     */
+    public $OnlyFilesChanged = 'git';
```

Comment:
This project use [camel case](https://en.wikipedia.org/wiki/CamelCase#Programming_and_coding). Please, can you change it to `onlyFilesChanged`?

And second think  in this line - why the class variable have value `git`? I don't understand to a meaning.